cap/modules/schemas/cli.py

```diff
@@ -25,16 +25,111 @@
 
 import json
 import os
+import re
 
 import click
 from flask_cli import with_appcontext
 from invenio_db import db
 from invenio_jsonschemas.errors import JSONSchemaNotFound
+from invenio_search import current_search_client
 from sqlalchemy.exc import IntegrityError
 
+from cap.utils import MutuallyExclusiveOption
+from cap.modules.deposit.errors import DepositValidationError
 from cap.modules.fixtures.cli import fixtures
+from cap.modules.records.api import CAPRecord
+from cap.modules.schemas.models import Schema
 
-from .models import Schema
+
+@fixtures.command()
+@click.option('--schema-url', '-u',
+              cls=MutuallyExclusiveOption,
+              mutually_exclusive=["ana_type"],
+              help='The schema that is used for validation.')
+@click.option('--ana-type', '-a',
+              cls=MutuallyExclusiveOption,
+              mutually_exclusive=["schema_url"],
+              help='The analysis type that is used for validation.')
+@click.option('--ana-version', '-v',
```

Comment:
make mutually exclusive also (not together with schema url - schema url/path has its own schema)..should only go together with `ana_type`, else get latest if not passed